api4/post.go

```diff
@@ -62,6 +62,8 @@ func createPost(c *Context, w http.ResponseWriter, r *http.Request) {
 		post.CreateAt = 0
 	}
 
+	c.App.SanitizeProps(post)
```

Comment:
I think I would be a bigger fan of putting the sanitization in `(*App).CreatePost` rather than each api handler.

Reply:
Noted. Move to `(*App).CreatePost` and also applied to `(*App).UpdatePost`